### PR TITLE
1.5x's slimes feeding rate

### PR DESCRIPTION
**Pull request by fen0s**
:cl: fen0s
balance: slimes are now 1.5x boosted at nutrition gain rate
/:cl:

[why]: Short rounds on Bagil server makes it impossible to get to "end-game" content of xenobio. 30 minutes is not enough to get some extracts like oil, while long rounds are rare thing on bagil. It's really hard to get some of extracts, so...

### CODE REVIEW

**Comment by fen0s** [comment]
stop why did it merge whole branch reeee github

**Comment by epochayur** [comment]
>30 minutes is not enough to get some extracts like oil...

sounds intended

**Comment by fen0s** [comment]
> > 30 minutes is not enough to get some extracts like oil...
> 
> sounds intended

i don't really think it is
sucks when shift ends already and you have just one blue slime and one orange

**Comment by ma44** [comment]
If it's because of bagil you may as well make a config for it

**Comment by ExcessiveUseOfCobblestone** [comment]
Play sybil/terry or go captain and recall if you dislike short rounds. 

> Config > hardcoded buff.

Admins should not be the arbiters of balance.

**Comment by fen0s** [comment]
> Config > hardcoded buff.

I'm not a coder, I'm a player. I dunno how to do config.

**Comment by Potato-Masher** [comment]
A good way to learn how to do configs is to look at recently merged configuration additions, if you feel inclined at all to make an effort to do it yourself. It's how I did it.

**Comment by 81Denton** [comment]
Downstreams can just not merge it

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -896,10 +896,10 @@
 	id = "nanite_hazard"
```
Comment (ma44) [inline code/modules/research/techweb/all_nodes.dm]: Pretty sure this is a undocumented change

**Comment by epochayur** [comment]
lol, thumbsdown for stealth changes

**Comment by fen0s** [comment]
i said that it merged the whole branch for some reason
i had failed nanites pr

**Comment by BebeYoshi** [comment]
Xenobiology is meant to be slow, because it can make overpowered things after a while with no problem.
This is just like making every plant grow faster but worse

**Comment by deathride58** [comment]
@81Denton The thing is, the issue OP is describing only exists on Bagil, which is currently the only TG-based server with a sub-40 minute average round time (Even Hippie's average round length is one hour, for crying out loud!). If this PR gets merged, then xenobio will be significantly overpowered on Sybil, Terry, and all of TG's downstreams.

Something that might actually be worth looking into is a config setting that adjusts timegated stuff like xenobio, RnD, material costs, etc, to a multiplier that represents the server's intended round length. This would make it very easy for server hosts/headmins to adjust timegated stuff to their server's average round length without touching a single line of code, making both the players and balance guys alike happy.

**Comment by fen0s** [comment]
Xenobio is meant to be slow, but it has giant RNG inside of it. Plants use RNG only with mutations, while you can get only grey slimes for good 20 minutes.

**Comment by fen0s** [comment]
> @81Denton The thing is, the issue OP is describing only exists on Bagil, which is currently the only TG-based server with a sub-40 minute average round time (Even Hippie's average round length is one hour, for crying out loud!). If this PR gets merged, then xenobio will be significantly overpowered on Sybil, Terry, and all of TG's downstreams.
> 
> Something that might actually be worth looking into is a config setting that adjusts timegated stuff like xenobio, RnD, material costs, etc, to a multiplier that represents the server's intended round length. This would make it very easy for server hosts/headmins to adjust timegated stuff to their server's average round length without touching a single line of code, making both the players and balance guys alike happy.

I'll repeat myself, I HAVE NO IEDA HOW TO CODE. Please, or fix this on bagil, or just merge that solution. It's quite impossible to get xenobio done right on bagil.

**Comment by deathride58** [comment]
Xenobio's RNG can be reacted to and planned for accordingly. Slime scanners give you all of the information you need to make educated decisions on which slimes you should keep around for a higher mutation chance. Xenobiology is extremely transparent with its RNG compared to other RNG-based mechanics. It only takes around 40 minutes from roundstart on average to reach Xenobio's "endgame" content so long as you know what you're doing unless you consistently have insanely bad luck, as the majority of your time is spent waiting for slimes to feed. That 40 minute soft timegate ensures that Xenobio reaches its "endgame" somewhere after the round's midpoint, where the majority of the server is aware of the roundtype, most departments have their shit together, and the patient antags start putting their plans into motion. However, on Bagil specifically, this poses a unique issue where Xenobio's "endgame" can only be achieved near or after the end of the round instead of a little bit after the round's midpoint as is intended, as the round's midpoint on Bagil is usually near the 20 minute mark instead of the 30 minute mark.

Making a handy config setting that allows server hosts/headmins to adjust timegates like Xenobio's soft timegates would be a good way to ensure that the time required for the timegates to be reached are adjusted appropriately depending on the server's average round length, as the 60 minute average round length isn't a one-size-fits-all solution. It'd make for an _excellent_ first coding project, as it'd only require adding a new config setting with the default value of "1", and making various timegated values be multiplied or divided accordingly depending on the value of that new config setting. It's something simple enough that it could be probably done in only five minutes by someone who only has basic knowledge of DM. If you need help, you can find the DM ref at http://www.byond.com/docs/ref/index.html , and you can join the #coderbus channel of the Rizon IRC server if you need additional help.

**Comment by ExcessiveUseOfCobblestone** [comment]
We keep our configs sync'd between servers and I don't like the idea of the admin-realm balancing the game.

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -896,10 +896,10 @@
 	id = "nanite_hazard"
 	display_name = "Hazard Nanite Programs"
 	description = "Extremely advanced Nanite programs with the potential of being extremely dangerous."
-	prereq_ids = list("nanite_harmonic", "alientech")
+	prereq_ids = list("nanite_harmonic", "syndicate_basic")
 	design_ids = list("spreading_nanites","mindcontrol_nanites","mitosis_nanites")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 10000)
-	export_price = 15000
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 15000)
+	export_price = 20000
```
Comment (ShizCalev) [inline code/modules/research/techweb/all_nodes.dm]: so are you going to remove this, orrrr?

Comment (fen0s) [reply code/modules/research/techweb/all_nodes.dm]: how do i? i have no idea how github works. and that's the reason this my old pr got here

**Comment by fen0s** [comment]
can now someone review it and merge/close

**Comment by SpaceManiac** [comment]
Declining this for now, we shouldn't be making changes to make up for the short rounds but instead making the rounds longer

**Pull request closed without merging**